chat_example/src/main/java/io/skygear/chatexample/ChatUsesAdapter.kt

```diff
@@ -38,6 +38,9 @@ class ChatUsesAdapter(currentUserId: String?) : RecyclerView.Adapter<ChatUsesAda
         val chatUser: ChatUser = mChatUsers[position]
 
         holder.idTv.text = chatUser.id
+        if (chatUser.record.get("username") != null) {
+            holder.idTv.text = chatUser.record.get("username").toString()
```

Comment:
I think we should fall back to use id if username is empty? As username for user is not required, for example for some app which are using email to login